daemon/remote/cmd/cmuxd-remote/agent_launch.go

```
@@ -146,6 +148,79 @@ func runOMORelay(socketPath string, args []string, refreshAddr func() string) in
+				return value
+			}
+		}
+	}
+	if value := strings.TrimSpace(os.Getenv("OPENCODE_PORT")); value != "" {
+		return value
+	}
+	listener, err := net.Listen("tcp", "127.0.0.1:0")
+	if err != nil {
+		return "4097"
+	}
+	defer listener.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `listener.Close` is not checked

(errcheck)